### PR TITLE
Ansible 2.0 compliant

### PR DESCRIPTION
**Pull request by awailly**
Partial support for ansible 2.0, we will be fully compliant with deprecated methods when we have a starting point with this PR.

### CODE REVIEW

**Comment by awailly** [comment]
@pchaigno Can you cross-check that everything is good? Seems acceptable for merging.

File: tests/callbacks/log_plays_coveralls.py.lol [hunk 1]
```diff
@@ -0,0 +1,206 @@
+# This file is part of Ansible
```
Comment (pchaigno) [inline tests/callbacks/log_plays_coveralls.py.lol]: What is this file? Looks like it was added on error...?

Comment (awailly) [inline tests/callbacks/log_plays_coveralls.py.lol]: No, this is the start of coveralls for ansible 2.0. I prefer to have it now as a skeleton and update after. Is it better to remove it?

Comment (pchaigno) [inline tests/callbacks/log_plays_coveralls.py.lol]: > Is it better to remove it?

I think it'd be cleaner to push it to another branch.

**Pull request merged**